src/pages/PointOfContact/index.tsx

```diff
@@ -116,52 +117,63 @@ function PointOfContact({ onSubmit }: PointOfContactProperties): JSX.Element {
-          const preFormattedData = getValues();
-          // 1.) Sending First Name and Last Name to the backend
-          const formattedUserProfileObject =
-            formatPointOfContactObject(preFormattedData);
-
-          await submitPointOfContact(auth, {
-            data: formattedUserProfileObject,
-            lei,
-            filingPeriod: year,
-          });
+        setIsSubmitting(true);
+        const preFormattedData = getValues();
+        // 1.) Sending First Name and Last Name to the backend
+        const formattedUserProfileObject =
+          formatPointOfContactObject(preFormattedData);
 
-          setIsLoading(false);
-        }
+        await submitPointOfContact(auth, {
+          data: formattedUserProfileObject,
+          lei,
+          filingPeriod: year,
+        });
 
         if (onSubmit) onSubmit(true);
       } catch (error) {
         // eslint-disable-next-line no-console
         console.log(error);
-        setIsLoading(false);
         if (onSubmit) onSubmit(false);
+      } finally {
+        setIsSubmitting(false);
       }
     } else {
       scrollToElement(formErrorHeaderId);
     }
   };
 
-  if (isFilingLoading)
-    return <LoadingContent message='Loading Filing data...' />;
+  // TODO: Redirect the user if the filing period or lei are not valid
+
+  if (isLoading) return <LoadingContent message='Loading Filing data...' />;
 
   return (
     <div id='point-of-contact'>
+      <FilingSteps />
       <FormWrapper>
         <FormHeaderWrapper>
+          <div className='mb-[0.9375rem]'>
+            <InstitutionHeading
+              eyebrow
+              name={institution?.name}
+              filingPeriod={year}
+            />
+          </div>
           <TextIntroduction
-            heading='Provide the point of contact'
-            subheading='Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation.'
+            heading='Provide point of contact'
+            subheading="Provide the name and business contact information of a person that the Bureau or other regulators may contact with questions about your financial institution's data submission."
             description={
               <Paragraph>
-                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-                eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
-                enim ad minim veniam, quis nostrud exercitation.
+                Your financial institution&apos;s point of contact information
+                will not be published with your financial institution&apos;s
+                data and pursuant to the rule will not be available to the
+                general public. This information is required pursuant to{' '}
+                <Link target='_blank' href={contactInfoLink}>
+                  § 1002.109(b)(3)
+                </Link>
```

Comment:
I've started a shared component for RegB links: https://github.com/cfpb/sbl-frontend/blob/main/src/components/CommonLinks.tsx#L85-L101